src/common/search/search-input.ts

```diff
@@ -47,14 +50,18 @@ class SearchInput extends LitElement {
         @value-changed=${this._filterInputChanged}
         .noLabelFloat=${this.noLabelFloat}
       >
-        <ha-icon icon="hass:magnify" slot="prefix" class="prefix"></ha-icon>
+        <ha-icon
+          icon="${this.hassio ? "hassio" : "hass"}:magnify"
```

Comment:
No need for this anymore 😆

Reply:
Noooooooooo, I just learned I should do it this way.... 🤣